Source/WebCore/css/CSSFontFaceSet.cpp

```diff
@@ -447,7 +447,7 @@ ExceptionOr<bool> CSSFontFaceSet::check(const String& font, const String& text)
     return true;
 }
 
-CSSSegmentedFontFace* CSSFontFaceSet::fontFace(FontSelectionRequest request, const AtomString& family)
+CSSSegmentedFontFace* CSSFontFaceSet::fontFace(const FontSelectionRequest& request, const AtomString& family)
```

Comment:
Not a large type, not obviously clear this is actually better?

Reply:
Will revert.